From 51e26ab5a3c2a46402be9e8ce21ac15d787c4890 Mon Sep 17 00:00:00 2001
From: Ren Jianing <jianing.ren@rock-chips.com>
Date: Tue, 10 Mar 2020 17:15:24 +0800
Subject: [PATCH] clk: rockchip: add clk_usbhost_utmi_ohci to critical for
 rv1126

The USB host controller needs four clocks, one more clock
clk_usb_host_utmi_ohci than other rockchip platform. However, the
upstream OHCI driver can only get up to three clocks. So the clock
is added to critical to guarantee host controller can work.

Change-Id: I8c1b15d69fd95c93e671ed0aaccc9d8da3c41aaf
Signed-off-by: Ren Jianing <jianing.ren@rock-chips.com>
---
 drivers/clk/rockchip/clk-rv1126.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rv1126.c b/drivers/clk/rockchip/clk-rv1126.c
index 3905a1debdca..b8f82b467fc5 100644
--- a/drivers/clk/rockchip/clk-rv1126.c
+++ b/drivers/clk/rockchip/clk-rv1126.c
@@ -1289,6 +1289,7 @@ static const char *const rv1126_cru_critical_clocks[] __initconst = {
 	"clk_ddrphy",
 	"pclk_pdddr",
 	"pclk_pdtop",
+	"clk_usbhost_utmi_ohci",
 };
 
 static void __iomem *rv1126_cru_base;
-- 
2.35.3

